src/file.cpp

```diff
@@ -198,10 +198,42 @@ read_file_result read_file(const char *path, posix_fd_file_ref file) {
 #endif
 
 #if defined(QLJS_FILE_WINDOWS)
+std::optional<std::wstring> mbstring_to_wstring(const char *mbstring) {
```

Comment:
Minor: I'd move this function (and also move the mbargv class) into their own module. This will let you (or anyone else) write tests for the code.